bin/convert-argv.js

```diff
@@ -233,21 +242,22 @@ module.exports = function(yargs, argv, convertOptions) {
 		}
 
 		function mapArgToBoolean(name, optionName) {
+			if (options[optionName || name]) {
+				options[name] = true;
+			}
+			//eslint-disable-next-line
+			if (name && options[name] == true) {
```

Comment:
Shouldn't be `===`?

Reply:
Implemented by webpack/core. You should just look at the test for now

Reply:
Also, let's keep it non-strict equal, cause it could be stringified